cypress-tests/cypress/support/commands.js

```diff
@@ -438,19 +484,31 @@ Cypress.Commands.add("refundCallTest", (requestBody, refund_amount, det, globalS
       "Content-Type": "application/json",
       "api-key": globalState.get("apiKey"),
     },
+    failOnStatusCode: false,
     body: requestBody
   }).then((response) => {
     logRequestId(response.headers['x-request-id']);
-
+    expect(res_data.status).to.equal(response.status);
     expect(response.headers["content-type"]).to.include("application/json");
-    globalState.set("refundId", response.body.refund_id);
-    expect(response.body.status).to.equal(det.refundStatus);
-    expect(response.body.amount).to.equal(refund_amount);
-    expect(response.body.payment_id).to.equal(payment_id);
+    
+    if(response.body.status !== undefined) {
```

Comment:
why are we having this condition? instead we could have response.body.status === "success" | "pending" right?

Reply:
status was being used to check whether the response was for 2xx, checking specifically for success and pending status would fail for 2xx which failed